templates/ts-crawlee-cheerio/src/main.ts

```diff
@@ -7,19 +7,32 @@
 // For more information, see https://docs.apify.com/sdk/js
 import { Actor } from 'apify';
 // For more information, see https://crawlee.dev
-import { CheerioCrawler } from 'crawlee';
-import { router } from './routes.js';
+import { CheerioCrawler, Dataset } from 'crawlee';
+
+interface Input {
+    startUrls: string[];
+    maxRequestsPerCrawl: number;
+}
 
 // Initialize the Apify SDK
 await Actor.init();
 
-const startUrls = ['https://apify.com'];
+const { startUrls, maxRequestsPerCrawl } = await Actor.getInputOrThrow<Input>();
```

Comment:
I just realized we don't have the INPUT key in the git repo, so running this locally will probably throw...

I guess let's do it the same way as in the JS template? You will need to type the `{}` too, otherwise it should look the same.

```suggestion
const { startUrls, maxRequestsPerCrawl } = await Actor.getInputOrThrow<Input>();

const {
    startUrls = ['https://crawlee.dev'],
    maxRequestsPerCrawl = 100,
} = await Actor.getInput<Input>() ?? {} as Input;
```